packages/plugin-ext/src/common/connection.ts

```diff
@@ -79,3 +87,10 @@ export class PluginWebSocketChannel implements IWebSocket {
         this.connection.dispose();
     }
 }
+
+/**
+ * Use `PluginChannel` instead.
+ *
+ * @deprecated since 1.19.0
+ */
+export const PluginWebSocketChannel = PluginChannel;
```

Comment:
I'm not sure we need a deprecation cycle here: it's not "official" API and I doubt someone actually used this implementation detai.

Reply:
I'm very fine with that, will remove.